src/test_binary_heap.py

```diff
@@ -3,23 +3,47 @@
 import pytest
 
 
-# @pytest.fixture(scope='function')
-# def bin_heap():
-#     """Create a fixture for testing."""
-#     from binary_heap import BinaryHeap
-#     return
-
-# def test_max_orient():
-#     """Test max organization works."""
-#     from binary_heap import BinaryHeap
-#     list_ = [1, 2, 3, 4, 1, 6, 7, 8]
-#     heap_instance = BinaryHeap(list_)
-#     assert heap_instance.heap == [8, 7, 6, 4, 1, 3, 2, 1]
-
-
-# def test_max_orient_5():
-#     """Test max organization works."""
-#     from binary_heap import BinaryHeap
-#     list_ = [1, 2, 3]
-#     heap_instance = BinaryHeap(list_)
-    # assert heap_instance.heap == [3, 2, 1]
\ No newline at end of file
+@pytest.fixture(scope='function')
+def binary_heap_object():
+    """Create a single item test."""
+    from binary_heap import BinaryHeap
+    return BinaryHeap()
+
+
+@pytest.fixture(scope='function')
+def big_binary_heap_object():
+    """Create a multiple level heap object."""
+    from binary_heap import BinaryHeap
+    obj = BinaryHeap()
+    for x in range(1, 9):
+        obj.push(x)
+    return obj
+
+
+def test_binary_heap_object(binary_heap_object):
+    """Test a single push."""
+    binary_heap_object.push(1)
+    assert binary_heap_object.heap == [1]
+
+
+def test_binary_heap_object_2(binary_heap_object):
+    """Test a single push."""
+    binary_heap_object.push(1)
+    binary_heap_object.push(2)
+    assert binary_heap_object.heap == [2, 1]
+
```

Comment:
It would be good to also test edge cases for `pop()` with 0, 1, and 2 items in the heap.